libmozevent/utils.py

```diff
@@ -21,21 +21,6 @@ class RunException(Exception):
     """
```

Comment:
You can also remove the RunException above.

Reply:
done :+1: